packages/editor/src/components/post-schedule/index.js

```diff
@@ -6,7 +6,7 @@ import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
 import { DateTimePicker } from '@wordpress/components';
 
-export function PostSchedule( { date, onUpdateDate } ) {
+export function PostSchedule( { date, modified, onUpdateDate } ) {
```

Comment:
One option here instead of adding new props to `<PostSchedule>` would be to simply use the current time down below. We just need to pass something other than `null` to `<DateTimePicker>` 
 in `currentDate` or else it acts up when the date has been cleared and you try to set another one. It immediately closes and uses the current time no matter what you click inside it.

Reply:
I got rid of this and moved to using current time instead

Reply:
Nevermind, has to be the `modified` time because it will get set `onBlur` and we want it to remain to be considered floating until it's specifically set to something else.